provider/terraform/custom_code.rb

```diff
@@ -96,6 +96,10 @@ class CustomCode < Api::Object
       # method's function header can't be changed, the template
       # inserts that for you - do not include it in your custom code.
       attr_reader :custom_delete
+      # This code replaces the entire create method.  Since the create
```

Comment:
As long as we're not using this, do you mind removing it + the changes in `resource.erb`? I'd rather not add unused code.